docs/conf.py

```diff
@@ -104,23 +104,16 @@
 
 # sphinx-version-warning config
 versionwarning_messages = {
-    'latest': (
-        'This document is for the development version. '
-        'For the stable version documentation, see <a href="/en/stable/">here</a>.'
-    ),
-    'stable': (
-        'This document is for the latest 3.x stable release. '
-        'For the 2.x documentation, see <a href="/en/2.x-line/">here</a>.'
-    ),
     '2.x-line': (
-        'This document is for the 2.x release branch. '
-        'For the 3.x documentation, see <a href="/en/stable/">here</a>.'
+        'marshmallow 2 will no longer be supported as of 2020-08-18. '
+        '<a href="https://marshmallow.readthedocs.io/en/latest/upgrading.html#upgrading-to-3-0">'
+        'Update your code to use marshmallow 3</a>.'
     ),
 }
 # Show warning at top of page
 versionwarning_body_selector = "div.document"
 # For debugging locally
-# versionwarning_project_version = 'latest'
+# versionwarning_project_version = '2.x-line'
```

Comment:
Not clear to me what this does.

I did read https://sphinx-version-warning.readthedocs.io/en/latest/configuration.html#confval-versionwarning_project_version but I don't get it.

Reply:
This is just for testing things locally. The warning banner doesn't show unless this is uncommented